code/DDSCodeTester.cpp

```diff
@@ -2535,6 +2535,23 @@ void dds_dataWriter_examples()
         //!--
     }
 
+    {
+        //DDS_CREATE_PAYLOAD_POOL_DATAWRITER
+        // A DataWriterQos must be provided to the creation method
+        DataWriterQos qos;
+
+        // Create PayloadPool
+        std::shared_ptr<CustomPayloadPool> payload_pool = std::make_shared<CustomPayloadPool>();
```

Comment:
I don't this this would compile since `CustomPayloadPool` is not defined as far as I can tell.